docker/templates/Dockerfile.tests.m4

```diff
@@ -1,91 +1,79 @@
+m4_include(`macros.m4')m4_dnl
```

Comment:
You can probably revert d92490b72f09548db639cfbf90e5fee36d4695c6 then.

Reply:
I will do so in the next pull request for switching to Perl 5.38 since there still are `Dockerfile.perl5.30.*`.